huggingface/pytorch/buildspec.yml

```diff
@@ -29,21 +102,21 @@ context:
       target: config.properties
 
 images:
-  BuildHuggingFacePytorchGpuPy38Cu113TrainingDockerImage:
-    <<: *TRAINING_REPOSITORY
-    build: &HUGGINGFACE_PYTORCH_GPU_TRAINING_PY3 false
-    image_size_baseline: &IMAGE_SIZE_BASELINE 15000
-    device_type: &DEVICE_TYPE gpu
-    python_version: &DOCKER_PYTHON_VERSION py3
-    tag_python_version: &TAG_PYTHON_VERSION py38
-    cuda_version: &CUDA_VERSION cu113
-    os_version: &OS_VERSION ubuntu20.04
-    transformers_version: &TRANSFORMERS_VERSION 4.17.0
-    datasets_version: &DATASETS_VERSION 1.18.4
-    tag: !join [ *VERSION, '-', 'transformers', *TRANSFORMERS_VERSION, '-', *DEVICE_TYPE, '-', *TAG_PYTHON_VERSION, '-',
-                 *CUDA_VERSION, '-', *OS_VERSION ]
-    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /,
-                         *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+  # BuildHuggingFacePytorchGpuPy37Cu111TrainingDockerImage:
```

Comment:
I see that `BuildHuggingFacePytorchGpuPy37Cu111TrainingDockerImage` has Py37 in it, can we change it to Py38 like: `BuildHuggingFacePytorchGpuPy38Cu111TrainingDockerImage`

Reply:
The Training Image won't be re-released. So it will stay commented out. The only reason for this PR is that we have to re-release the 1.9.1 DLCs since there was a bug in the `sagemaker-infernece` package. @arjkesh told me back then that we cannot re-release a specific `buildspec-xxx.yaml` so i should open a PR and adjust the `buildspec.yaml` to only include the changes to then afterwards revert it to the correct one. If that's wrong please let me know